console/console-init/ui/src/modules/address-detail/AddressDetailPage.tsx

```diff
@@ -45,7 +45,7 @@ export default function AddressDetailPage() {
         <BreadcrumbItem>
           <Link
             id="ad-page-link-addresses"
-            to={`/address-spaces/${namespace}/${name}/${type}/addresses`}
+            to={`/messaging-projects/${namespace}/${name}/${type}/addresses`}
```

Comment:
I would like to suggest create a routePath varibale for route and use that as we have same route at mutiple plcaes in this page. It would help if any changes reuire in future then yiu can make chnage at single place.
const routePath=`/messaging-projects/${namespace}/${name}/${type}/addresses`